test/src/test-capi-consolidation-plan.cc

```diff
@@ -109,35 +105,20 @@ void ConsolidationPlanFx::create_sparse_array(bool allows_dups, bool encrypt) {
   filter_list.add_filter(filter);
   schema.set_coords_filter_list(filter_list);
 
-  if (encrypt) {
-    Array::create(array_name_, schema, enc_type_, key_);
-  } else {
-    Array::create(array_name_, schema);
-  }
+  Array::create(array_name_, schema);
```

Comment:
do we really want to remove coverage for encypted arrays in those tests or just change the way we create encrypted arrays?

Reply:
explained offline that these were just copy paste leftovers in tests.